config/config.go

```diff
@@ -45,9 +46,12 @@ func (c *Config) MakeAPICall(verb string, endpoint string, body []byte) (*http.R
 	var request *http.Request
 	var reqErr error
 
-	url := c.Endpoint + endpoint
+	url, err := url.JoinPath(c.Endpoint, endpoint)
```

Comment:
Looks good, but wasn't sure it cleaned everything prior to join so tested here in case and it does: https://go.dev/play/p/skIAoIOqKZ8